2005-01-20  Tollef Fog Heen  <tfheen@debian.org> 

	* Makeconfig: add multiarch support

--- glibc-2.3.6/Makeconfig	2006-02-22 12:44:52.424091146 +0100
+++ glibc-2.3.6/Makeconfig	2006-02-22 12:45:52.161604171 +0100
@@ -176,6 +176,11 @@
 endif
 inst_libdir = $(install_root)$(libdir)
 
+# Extra places to look for libraries
+ifndef extra_libdir
+extra_libdir = /lib/$(config-machine)-$(config-os):$(exec_prefix)/lib/$(config-machine)-$(config-os)
+endif
+
 # Where to install the shared library and dynamic linker.
 ifndef slibdir
 slibdir = $(exec_prefix)/lib
@@ -479,6 +484,10 @@
 default-rpath = $(libdir)
 endif
 
+ifdef extra_libdir
+default-rpath += :$(extra_libdir)
+endif
+
 ifndef link-extra-libs
 link-extra-libs = $(LDLIBS-$(@F))
 link-extra-libs-static = $(link-extra-libs)
